packages/evm/test/disputes/disputes-evidence.js

```diff
@@ -130,14 +129,8 @@ contract('DisputeManager', ([_, juror500, juror1000, juror1500]) => {
     })
 
     context('when the sender is not the arbitrable of the dispute', () => {
-      let fakeArbitrable
-
-      beforeEach('mock non arbitrable', async () => {
-        fakeArbitrable = await FakeArbitrable.new(court.address)
-      })
-
       it('reverts', async () => {
```

Comment:
Theoretically we should be able to submit evidence with `fakeArbitrable` if we set it as the dispute subject?

Reply:
Addressed in d042bfc9e25617d667573d7efc824159c25502c9